include/swift/DependencyScan/SerializedModuleDependencyCacheFormat.h

```diff
@@ -124,6 +124,7 @@ using ModuleInfoLayout =
                    IdentifierIDField,            // moduleName
                    ContextHashIDField,           // contextHash
                    ImportArrayIDField,           // moduleImports
+                   ImportArrayIDField,           // optionalModuleImports
                    DependencyIDArrayIDField      // resolvedDirectModuleDependencies
```

Comment:
Does this need a version bump to the format?

Reply:
Done, thanks!